packages/react-components/src/components/VideoGallery/LocalScreenShare.tsx

```diff
@@ -48,12 +48,13 @@ export const LocalScreenShare = React.memo((props: { localParticipant: VideoGall
     </Stack>
   );
 
+  const displayName =
+    localParticipant?.displayName === '' || !localParticipant?.displayName
```

Comment:
nit, this can just be `!localParticipant?.displayName` (no need for empty string check)
```suggestion
  !localParticipant?.displayName
```